tests_recording/test_api.py

```diff
@@ -33,9 +32,8 @@ def setUp(self):
 
     def check_version_increase(self):
         # change specfile little bit to have there some change
-        specfile_location = os.path.join(self.lp.working_dir, "python-ogr.spec")
-        with open(specfile_location, "r") as myfile:
-            filedata = myfile.read()
+        specfile_location = Path(self.lp.working_dir, "python-ogr.spec")
```

Comment:
We can use also this syntax:

```suggestion
        specfile_location = self.lp.working_dir / "python-ogr.spec")
```

(There are multiple occurrences, where you can use this syntax.